packages/core/src/Actor.ts

```diff
@@ -154,7 +154,7 @@ export class Actor<TEvent extends EventObject, TEmitted> {
   public current: TEmitted;
```

Comment:
should we rename all `TEmitter` to `TSnapshot`? 🤔

Reply:
No because emitted !== snapshot (sometimes, but not always)